code/modules/reagents/chemistry/reagents/food.dm

```diff
@@ -43,19 +44,22 @@
 	id = "protein"
 	description = "Various essential proteins and fats commonly found in animal flesh and blood."
 	diet_flags = DIET_CARN | DIET_OMNI
+	taste_message = "meat"
 
 /datum/reagent/consumable/nutriment/plantmatter		// Plant-based biomatter, digestable by herbivores and omnivores, worthless to carnivores
 	name = "Plant-matter"
 	id = "plantmatter"
 	description = "Vitamin-rich fibers and natural sugars commonly found in fresh produce."
 	diet_flags = DIET_HERB | DIET_OMNI
+	taste_message = "plant matter"
 
 /datum/reagent/consumable/vitamin
 	name = "Vitamin"
 	id = "vitamin"
 	description = "All the best vitamins, minerals, and carbohydrates the body needs in pure form."
 	reagent_state = SOLID
 	color = "#664330" // rgb: 102, 67, 48
+	taste_message = null
```

Comment:
I'd rather this was something really clear and descriptive OR wasn't there at all, considering how useful they are. I really don't like when you can meta stuff based on some esoteric knowledge of how suddenly tasting Flintstones gives you a more efficient metabolic system. 😉